src/types/global.d.ts

```diff
@@ -4,5 +4,8 @@ declare global {
   interface Window {
     adobeDataLayer: Record<string, any>[];
     gtag: (...args: any[]) => void;
+    newrelic: {
+      noticeError: any;
```

Comment:
nit: can/should we type this is as function?

Reply:
I like it! Updated.